receiver/filelogreceiver/README.md

```diff
@@ -19,35 +19,35 @@ Tails and parses logs from files.
 |-------------------------------------|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `include`                           | required                             | A list of file glob patterns that match the file paths to be read                                                                                                                                                                                               |
-| `exclude`                           | []                                   | A list of file glob patterns to exclude from reading                                                                                                                                                                                                            |
-| `start_at`                          | `end`                                | At startup, where to start reading logs from the file. Options are `beginning` or `end`                                                                                                                                                                         |
-| `multiline`                         |                                      | A `multiline` configuration block. See below for more details                                                                                                                                                                                                   |
-| `force_flush_period`                | `500ms`                              | Time since last read of data from file, after which currently buffered log should be send to pipeline. Takes `time.Duration` (e.g. `10s`, `1m`, or `500ms`) as value. Zero means waiting for new data forever                                                   |
-| `encoding`                          | `utf-8`                              | The encoding of the file being read. See the list of supported encodings below for available options                                                                                                                                                            |
+| `include`                           | required                             | A list of file glob patterns that match the file paths to be read. The file path refers to the file path in the docker container (if you want to use a file from your local computer, you need to use a [docker volume](https://docs.docker.com/storage/volumes/)).                                                                                                                                                                                             |
```

Comment:
The collect will will not always run inside a container. I think we can omit this comment as about the file path inside the container as this has to do with the system and is general knowledge when using containers, you need to mount a volume with a path to the host if you need access to it.